README.md

```diff
@@ -73,9 +73,9 @@ For example:
 
 ## Sync API logging
 
-The Sync API logs messages at debug level, except in error scenarios
- where additional context is available that can't easily be passed to
- the calling code.  This idea comes from the
- [level suggestions for libraries in the Bunyan project](https://github.com/trentm/node-bunyan#level-suggestions).
- Users of the Sync API can then enable debug-level logging if they
- would like to see more output.
+Users of the Sync API can then enable debug-level logging if they would like to see more output. This is useful for debugging purposes.
+It's possible to pass environment variables to enable the logging according the rules specified for [debug](https://www.npmjs.com/package/debug) module:
+  
+```
+DEBUG="fh-mbaas-api:sync" DEBUG_COLORS=1 ./yourscript 
```

Comment:
Good call 👍  I can see people getting annoyed it they can't see any clear way to enable them.